### ISSUE TITLE
Difference between having Flash set to activated/never activate

### ISSUE THREAD

**Issue by JKAbrams** [ChrisAntaki/plugins-and-mimetypes-firefox#4]
Curtsy to about:robots comment on the add-ons page:

What I expect from this plugin, combined with setting Flash to "Aks to activate" is that there should be no way for a website to know that I have flash installed unless I click "activate" on a certain page. This, as about:robots found, is not the case for http://www.tagesschau.de , which I can confirm.

When you set Flash to "Never activate" the page serves HTML5 video, which is what we want, but if you set the Flash to "Ask to activate" and use this plugin it will still serve flash video.

Basically,  what we would really like is to have HTML5 be used by default, and not even tell web-pages that Flash is installed unless we know we really need it and make a special exception for that site (i.e. by pressing "Allow now" for that certain site).
This is the only way to slowly kill Flash while in the meantime be able to use the sites that still depend on it, and the reason I really want your plugin (aside from the nice addition reducing a few bits of tracking information).

**Comment by ChrisAntaki** [comment]
Sadly, it looks like Mozilla broke the privacy feature that made this plugin possible. https://bugzilla.mozilla.org/show_bug.cgi?id=1169945
